.github/workflows/ci-e2e.yaml

```
@@ -20,12 +20,12 @@ jobs:
     name: Integration tests with MySQL
     strategy:
       matrix:
-        mysql-version: ["8.0.28", "8.0.36", "8.0.37", "8.4.0"]
+        mysql-version: ["8.0.28", "8.0.36", "8.0.37", "8.4.0", "8.4.2"]
```

Isn't our policy support only the latest LTS release?
I mean, we do not need to keep `8.4.0` here.
```suggestion
        mysql-version: ["8.0.28", "8.0.36", "8.0.37", "8.4.2"]
```

---

Also write `8.0.39`